lib/fork.js

```diff
@@ -18,13 +18,29 @@ module.exports = function (args) {
 
 	var ps = childProcess.fork(babel, args, options);
 
+	var killed = false;
+	var killedPromise = new Promise(function (resolve) {
```

Comment:
Apart from the discussion if this is necessary or not, I think this can be written with [rfpify](https://github.com/SamVerschueren/rfpify).

```js
var killedPromise = rfpify(ps.on.bind(ps), Promise)('exit').then(function (code) {
    killed = true;
    return code;
});
```